src/components/Filters/FilterModal.tsx

```diff
@@ -197,8 +197,11 @@ const FilterInputs = ({
 	const [searchTerm, setSearchTerm] = React.useState('');
 	const debouncedSetSearchTerm = debounce(setSearchTerm, 300);
 	const operators = React.useMemo(
-		() => (signature.field ? getOperators(schema, signature.field) : []),
-		[signature.field, schema],
+		() =>
+			signature.field
+				? getOperators(schema, signature.field, signature.refScheme)
```

Comment:
Does this also work or `x-foreign-key-scheme` based properties?

Reply:
yes, because at this moment of the code we created an item for each schema.properties and each x-ref-scheme || x-foreign-key-scheme